pyvista/plotting/prop3d.py

```diff
@@ -213,6 +223,19 @@ def orientation(self) -> tuple:  # numpydoc ignore=RT01
     def orientation(self, value: tuple):  # numpydoc ignore=GL08
         self.SetOrientation(value)
 
+    @property
+    def origin(self) -> tuple:  # numpydoc ignore=RT01
+        """Return or set the entity origin.
+
+        This is the point about which all rotations take place.
+
+        """
+        return self.GetOrigin()
+
+    @origin.setter
+    def origin(self, value: tuple):  # numpydoc ignore=GL08
```

Comment:
```suggestion
    def origin(self, value: Vector):  # numpydoc ignore=GL08
```

Reply:
This change to the typing has been made in 7aaaa1e0